spec/conf.py

```diff
@@ -62,6 +62,8 @@
     ('py:class', 'collections.abc.Sequence'),
     ('py:class', "Optional[Union[int, float, Literal[inf, - inf, 'fro', 'nuc']]]"),
     ('py:class', "Union[int, float, Literal[inf, - inf]]"),
+    ('py:obj', "typing.Optional[typing.Union[int, float, typing.Literal[inf, - inf, 'fro', 'nuc']]]"),
+    ('py:obj', "typing.Union[int, float, typing.Literal[inf, - inf]]"),
```

Comment:
This looks like it's going to be a continuous source of friction. Not a blocker for this PR, but can't we just turn off this cross-linking?